test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest>=6.2.5
+pytest-spark>=0.6.0
+pytest-dbt-core==0.1.0.dev2
```

Comment:
I moved all the set-up logic into a [package](https://pypi.org/project/pytest-dbt-core/)